src/codeu/chat/client/ClientConversation.java

```diff
@@ -27,165 +27,169 @@
 
 public final class ClientConversation {
```

Comment:
Revert all changes to this file since there only appears to be whitespace changes.